blue86321/student-voting-portal
# Feat/db/dump

## Title
Prepare database sample data

## Summary
- prepare sample data, dump database [feat: mysql dump data](https://github.com/blue86321/student-voting-portal/commit/3637fb40e24ce2a9b0c56e10322e52877a6273d0)
- backend: fix `user` filter and vote validator
  - [fix: backend user filter](https://github.com/blue86321/student-voting-portal/commit/73d14da8a240c8fb1eacb7792faf2c3a65f3fcbf)
  - [feat: backend vote validator: position and candidate](https://github.com/blue86321/student-voting-portal/commit/9c2f4336414ee7eb32be34d78a4f6eb12a43dc82)
- frontend: fix install error and case-sensitive problem
  - [fix: frontend docker, component case](https://github.com/blue86321/student-voting-portal/commit/c8a748ad610a4b82ed118618dd48ed1568d8d001)
  - [fix: frontend case sensitive](https://github.com/blue86321/student-voting-portal/commit/8af3bde9ffb17dda5aeb371e598e2f43719444f4)
- fix `.env.example` typo [fix: typo in .env.example](https://github.com/blue86321/student-voting-portal/commit/5d90eed0d62c8441c20ec402268cb16ef3c79c9c)


## Test Plan
```shell
docker compose up
```
- visit http://localhost:8000 to check backend data
- visit http://localhost:3000 to check frontend page
